stat/testuv/norm_test.go

```diff
@@ -0,0 +1,119 @@
+package testuv
+
+import (
+	"math"
+	"testing"
+
+	"golang.org/x/exp/rand"
+	"gonum.org/v1/gonum/stat/distuv"
+)
+
+func getNormalValues() []float64 {
+	normal := distuv.Normal{Mu: 0, Sigma: 1, Src: rand.New(rand.NewSource(1))}
+	size := 100
+	nums := make([]float64, size)
+	for i := range nums {
+		nums[i] = normal.Rand()
+	}
+	return nums
+}
+
+func TestNormalSkewTest(t *testing.T) {
+	// Even though the skew test and kurtosis test are a copy from the scipy's the results
+	// are not exactly the same because the skew/kurtosis corrections are not exactly the same
+	// but the statistic value is close enough.
+	for i, test := range []struct {
+		values   []float64
+		expected float64
+	}{
+		{
+			values:   []float64{0, 1, 2, 3, 4, 5, 6, 7, 8},
+			expected: 1.018464355396213,
+		},
+		{
+			values:   []float64{2, 8, 0, 4, 1, 9, 9, 0},
+			expected: 0.5562585562766172,
+		},
+		{
+			values:   []float64{1, 2, 3, 4, 5, 6, 7, 8000},
+			expected: 4.319816401673864,
+		},
+		{
+			values:   []float64{100, 100, 100, 100, 100, 100, 100, 101},
+			expected: 4.319820025201098,
+		},
+	} {
+		z := NormalSkewTest(test.values)
+		if math.Abs(z-test.expected) > 1e-7 {
```

Comment:
`if floats.EqualWithinAbsOrRel(z, test.want, tol, tol) ...` and define a reasonable `const tol`.